dvc/repo/plots/template.py

```diff
@@ -574,24 +167,19 @@ def init(self):
         from dvc.utils.fs import makedirs
 
         makedirs(self.templates_dir, exist_ok=True)
-        for t in self.TEMPLATES:
-            self._dump(t())
-
-    def _dump(self, template):
-        path = os.path.join(self.templates_dir, template.filename)
-        with open(path, "w") as fd:
-            fd.write(template.content)
-
-    def load(self, name):
-        try:
-            path = self.get_template(name)
-
-            with open(path) as fd:
-                content = fd.read()
-
-            return Template(content, name=name)
-        except TemplateNotFoundError:
-            for template in self.TEMPLATES:
-                if template.DEFAULT_NAME == name:
-                    return template()
-            raise
+        import pkg_resources  # type: ignore
+
+        if pkg_resources.resource_exists(
+            __name__, PlotTemplates.PKG_TEMPLATES_DIR
+        ):
+            templates = pkg_resources.resource_listdir(
+                __name__, PlotTemplates.PKG_TEMPLATES_DIR
+            )
+            for template in templates:
+                content = pkg_resources.resource_string(
+                    __name__, f"{PlotTemplates.PKG_TEMPLATES_DIR}/{template}"
+                )
+                with open(
+                    os.path.join(self.templates_dir, template), "wb"
+                ) as fd:
+                    fd.write(content)
```

Comment:
Preserving previous behaviour of writing default templates to `.dvc/plots`. It will be removed in next change.